apps/state/lib/state/stops_on_route.ex

```diff
@@ -187,6 +194,20 @@ defmodule State.StopsOnRoute do
     Enum.filter(global_order, &MapSet.member?(group_stop_id_map, &1))
   end
 
+  # additional ordered stop IDs to include during the merge
+  @spec stop_order_overrides(Model.Route.id(), Model.Direction.id()) :: stop_id_list
```

Comment:
This should return `:: [stop_id_list]`. It's not obvious just from the name.

Reply:
Also not obvious to Dialyzer since it didn't find any issues :) Fixed.